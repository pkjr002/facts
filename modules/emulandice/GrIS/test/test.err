Installing package from './emulandice_1.1.0.tar.gz'
Installing emulandice
'/projects/community/R/3.6.3/gc563/lib64/R/bin/R' --vanilla CMD INSTALL  \
  --preclean '/tmp/Rtmpjnn85q/packrat-install-37be50b7e274/emulandice'  \
  --library='/scratch/rk509/test.1668004635/packrat/lib/x86_64-pc-linux-gnu/3.6.3'  \
  --install-tests 

* installing *source* package ‘emulandice’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (emulandice)
Loading required package: ggplot2
Loading required package: dplyr

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Loading required package: tidyr
Loading required package: readr
Loading required package: purrr
Loading required package: tibble
Loading required package: stringr
Loading required package: forcats
emulandice_steer.sh: line 16: 14463 Terminated              Rscript -e "source('packrat/init.R')" -e "library(emulandice)" -e "emulandice::main('decades',dataset='$emulandice_dataset',N_FACTS=$nsamps,ice_sources=c('$3'))"
