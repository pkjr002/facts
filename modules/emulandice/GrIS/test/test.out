Initiating test in /scratch/rk509/test.1668004635...

Extracting data files...

Executing workflow...
source emulandice_environment.sh --Rscript
[1] "____________________________________________________"
[1] "Welcome to emulandice"
[1]  
[1] "Can ignore the warnings from optimize"
[1] "____________________________________________________"
python emulandice_preprocess.py --pipeline_id GrIS-emulandice-GrIS-ssp585 --baseyear 2005 --input_data_file test/input/fair-temperature-fair-ssp585_gsat.nc
python emulandice_fit_GrIS.py --pipeline_id GrIS-emulandice-GrIS-ssp585
python emulandice_project_GrIS.py --pipeline_id GrIS-emulandice-GrIS-ssp585
Rscript -e source('packrat/init.R') -e library(emulandice) -e emulandice::main('decades',dataset='FACTS_CLIMATE_FORCING.csv',N_FACTS=50,ice_sources=c('GrIS'))
python emulandice_postprocess_GrIS.py --pipeline_id GrIS-emulandice-GrIS-ssp585 --locationfile /home/rk509/facts/modules/emulandice/GrIS/test/../../../../scripts/moduletest/location_list.lst --chunksize 600

Collecting output files...
GrIS-emulandice-GrIS-ssp585_globalsl.nc
GrIS-emulandice-GrIS-ssp585_localsl.nc

Cleaning up...
